examples/tutorials/Introduction_to_Equivariance.ipynb

```diff
@@ -7,21 +7,21 @@
     "tags": []
```

Comment:
<p>Notes: </p><ul><li>Let's revert to equivariant modeling</li><li>Let's add a reference to Michael White's book to give proper attribution</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchem/deepchem/pull/4090/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>